XamMacSwiftUITest/AppDelegate.cs

```diff
@@ -12,15 +12,20 @@ public class AppDelegate : NSApplicationDelegate
 	{
 		public override void DidFinishLaunching (NSNotification notification)
 		{
-			var window = new NSWindow (new CGRect (x: 0, y: 0, width: 480, height: 300),
-				NSWindowStyle.Titled | NSWindowStyle.Closable | NSWindowStyle.Miniaturizable | NSWindowStyle.Resizable | NSWindowStyle.FullSizeContentView,
-				NSBackingStore.Buffered, deferCreation: false);
+			var flags = NSWindowStyle.Titled | NSWindowStyle.Closable | NSWindowStyle.Miniaturizable | NSWindowStyle.Resizable | NSWindowStyle.FullSizeContentView;
+			var window = new NSWindow (new CGRect (x: 0, y: 0, width: 480, height: 300), flags, NSBackingStore.Buffered, deferCreation: false);
 
 			window.Center ();
 
 			window.ContentView = NSHostingView.Create (new ClickButton());
 
 			window.MakeKeyAndOrderFront (null);
 		}
-	}
+
```

Comment:
Made this more consistent with the F# project.

Reply:
Love it!